mil_perception/src/mil_vision_lib/cv_tools.cpp

```diff
@@ -0,0 +1,70 @@
+#include <mil_vision_lib/cv_tools.hpp>
+
+bool mil_vision::get3DPointsFromParam(std::string topic, std::vector<cv::Point3f>& points)
+{
+    XmlRpc::XmlRpcValue list;
```

Comment:
It would be better to use a std::list here. It's the more standard way to do it now and it's a much more simple implementation.